src/oneview.js

```diff
@@ -40,15 +40,13 @@ const main = (robot) => {
 
   robot.logger.info('Initializing OneView');
   const client = new ovClient(oneviewConfig, robot);
-  client.login({
-    'userName': oneviewConfig.userName,
-    'password': oneviewConfig.password
-  }, false).then(() => {
-    robot.logger.info('Logged into OV appliance.');
+
+  client.login(false).then(() => {
+    robot.logger.info('Logged into OV appliance(s).');
```

Comment:
OV --> OneView.
Also, would be nice to say "appliance" or "appliances" vs. "appliance(s)"